RevenueCat/Scripts/PurchasesWrapperiOS.cs

```diff
@@ -2,12 +2,14 @@
 using System.Runtime.InteropServices;
 using UnityEngine;
 
-#if UNITY_IOS
+// #if UNITY_IOS
 public class PurchasesWrapperiOS : IPurchasesWrapper
 {
     [DllImport("__Internal")]
     private static extern void _RCSetupPurchases(string gameObject, string apiKey, string appUserId, bool observerMode, string userDefaultsSuiteName);
-    public void Setup(string gameObject, string apiKey, string appUserId, bool observerMode, string userDefaultsSuiteName)
+
+    public void Setup(string gameObject, string apiKey, string appUserId, bool observerMode,
+        string userDefaultsSuiteName, bool useAmazon, string dangerousSettingsJson)
```

Comment:
slight departure from the other amazon PRs here - I'm cutting off the params at this layer, and making no-op methods for stuff that doesn't apply here too, for simplicity.